### DISCUSSION TITLE
安装问题！命令一键安装出错后！我问了一下ai！ai都知道1panel安装错误大多卡在无法安装Docker上！优秀的你们不知道吗？

### DISCUSSION THREAD

**Discussion by kinggodman** [1Panel-dev/1Panel#6312] [Q&A]
### 1Panel 版本

1

### 请描述您的需求或者改进建议

ai都知道1panel安装错误大多卡在无法安装Docker上！优秀的你们不知道吗？
   这种入门门槛难道是想让更少人使用1panl    我们国人的产品在入门上就输给老外？
     我觉得你们这些优秀的人可以更好的完善这个问题！ 尽量做的轻便、docker占用资源本来也蛮大不适合羽量级服务器！这就大大增加了   我国穷学生 新手程序员的使用门槛！   望诸位优秀的人可以做的更好！越来越幸福

### 请描述你建议的实现方案

1、找出替代docker的备用产品可以安装时不出错！
2、用更节约资源的方式取代docker的功能！
3、借鉴别家开源界面的代码解题思路解决安装门槛！（1、安装不上有备选替代链接 2、尽量实现羽量级轻便）  我们也应该华丽花哨的不要学老外的轻便  只设置主要功能 给用户更多轻便！  增加盈利的服务 可以加载服务之下 真正有用实用的 产品 不用显眼！只要 周道干净 ！写清楚干嘛用的更贴心一些  很多人都会买单！  这样相信不多长时间 你们就可以 让宝塔 逐步从市场上消失

**Comment by wanghe-fit2cloud** [comment]
不至于去问 AI，docker 失败这类错误信息都是有的，目前错误后也会提示去下载离线包安装。
docker 在线安装脚本也有增加可用的源

**Comment by kinggodman** [comment]
不瞒你说~~！ 我从ubuntu换到centos！ ~~！还把centos的yum整了vauit的yum~~！  然后打底整了几十遍没安装成功！搞两天了！你说郁闷不

**Reply by wanghe-fit2cloud** [reply]
> 不瞒你说~~！ 我从ubuntu换到centos！ ~~！还把centos的yum整了vauit的yum~~！ 然后打底整了几十遍没安装成功！搞两天了！你说郁闷不

这种情况，就可以直接根据错误提示信息，来选择离线包安装。
> 离线安装文档地址：https://1panel.cn/docs/installation/package_installation/

**Reply by kinggodman** [reply]
1Panel 离线安装包  开源的离线安装包！为什么联网说连接超时！！郁闷！加群了！群里请教吧！ 我是下的github上的离线安装本！  没手机号~~！所有没能在你发的链接下载成功！

**Reply by kinggodman** [reply]
~~!搞定了 又换了几遍系统!改了几次源！  而且docker也下载了几个版本！ 要Docker ce的哪个才行

**Reply by kinggodman** [reply]
普通版本的docker！ 在服务器界面下载启动工具的时候会出错

**Comment by kinggodman** [comment]
好的我试试！辛苦你了~~！


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年8月30日 22:07 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [1Panel-dev/1Panel] 安装问题！命令一键安装出错后！我问了一下ai！ai都知道1panel安装错误大多卡在无法安装Docker上！优秀的你们不知道吗？ (Discussion #6312) |

不瞒你说~~！ 我从ubuntu换到centos！ ！还把centos的yum整了vauit的yum！ 然后打底整了几十遍没安装成功！搞两天了！你说郁闷不

这种情况，就可以直接根据错误提示信息，来选择离线包安装。

离线安装文档地址：https://1panel.cn/docs/installation/package_installation/

**Comment by kinggodman** [comment]
~~！还要写手机号 才能下载！






---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年8月30日 22:07 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [1Panel-dev/1Panel] 安装问题！命令一键安装出错后！我问了一下ai！ai都知道1panel安装错误大多卡在无法安装Docker上！优秀的你们不知道吗？ (Discussion #6312) |

不瞒你说~~！ 我从ubuntu换到centos！ ！还把centos的yum整了vauit的yum！ 然后打底整了几十遍没安装成功！搞两天了！你说郁闷不

这种情况，就可以直接根据错误提示信息，来选择离线包安装。

离线安装文档地址：https://1panel.cn/docs/installation/package_installation/

**Comment by kinggodman** [comment]
github上~~！可以  谢谢辛苦你了






---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年8月30日 22:07 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [1Panel-dev/1Panel] 安装问题！命令一键安装出错后！我问了一下ai！ai都知道1panel安装错误大多卡在无法安装Docker上！优秀的你们不知道吗？ (Discussion #6312) |

不瞒你说~~！ 我从ubuntu换到centos！ ！还把centos的yum整了vauit的yum！ 然后打底整了几十遍没安装成功！搞两天了！你说郁闷不

这种情况，就可以直接根据错误提示信息，来选择离线包安装。

离线安装文档地址：https://1panel.cn/docs/installation/package_installation/

**Comment by kinggodman** [comment]
加群了！  机器人还没通过！  哈哈哈！下载github上的离线包 运行SH脚本失败！连接超时！




---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年8月30日 22:07 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [1Panel-dev/1Panel] 安装问题！命令一键安装出错后！我问了一下ai！ai都知道1panel安装错误大多卡在无法安装Docker上！优秀的你们不知道吗？ (Discussion #6312) |

不瞒你说~~！ 我从ubuntu换到centos！ ！还把centos的yum整了vauit的yum！ 然后打底整了几十遍没安装成功！搞两天了！你说郁闷不

这种情况，就可以直接根据错误提示信息，来选择离线包安装。

离线安装文档地址：https://1panel.cn/docs/installation/package_installation/

**Comment by Anyexyz** [comment]
没有日志的情况下，无异于闭眼开车。

**Reply by kinggodman** [reply]
行吧！谢谢你了
